test/test_repolib.py

```diff
@@ -31,9 +32,19 @@
 import os
 from iniparse import ConfigParser
 
+# repofile must be patched and reloaded to import AptRepofile, otherwise
+# the class is not defined in the first place
+from subscription_manager import repofile
+deb_mock = MagicMock()
+deb_mock.Deb822 = dict
+with patch.dict("subscription_manager.repofile.modules", {"debian.deb822": deb_mock}):
+    six.moves.reload_module(repofile)
```

Comment:
since sub-man is py3-only, better avoid `six` in new code

Reply:
Removed as suggested. I'm using `importlib.reload` now.